tests/_support/Step/Acceptance/Installation/Installation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Step\Acceptance\Installation;
+
+use Page\Acceptance\Administrator\LoginPage;
+use Page\Acceptance\Administrator\ControlPanelPage;
+
+class Installation extends \AcceptanceTester
+{
+	/**
+	 * @Given Joomla CMS is installed
+	 */
+	public function joomlaCMSIsInstalled()
+	{
+		$I = $this;
+		// @todo: the following path should be taken from configuration and not hardcoded as it is now.
+		$I->assertFileExists('tests/joomla-cms3/configuration.php');
```

Comment:
How does it look now?
https://github.com/joomla-projects/gsoc16_browser-automated-tests/pull/42/commits/f907231a9b144dfba7a8392c42b1cb1440b476e4#r69120492

https://github.com/joomla-projects/gsoc16_browser-automated-tests/pull/42/commits/f907231a9b144dfba7a8392c42b1cb1440b476e4#r69120656

I have removed the Given, because I didn't found anything valuable to set as given. Thoughts?